queries.sql

```diff
@@ -1,3 +1,85 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+
+/* first query, to select all names ending in mon */
+
+vet_clinic=# SELECT * FROM animals WHERE name LIKE '%mon';
+ id |  name   | date_of_birth | escape_attempts | neutered | weight_kg
+----+---------+---------------+-----------------+----------+-----------
+  1 | Agumon  | 2020-02-03    |               0 | t        |     10.23
+  2 | Gabumon | 2018-11-15    |               2 | t        |      8.00
+  4 | Davimon | 2017-05-12    |               5 | t        |     11.00
+(3 rows)
+
+/* second query, selects names of animals between given dates */
+
+vet_clinic=# SELECT name FROM animals
+vet_clinic-# WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+  name
+---------
+ Gabumon
+ Davimon
+(2 rows)
+
+/* third query , select names of animals who are neutered and have escape attempts less than 3 */
+
+vet_clinic=# SELECT name FROM animals
+vet_clinic-# WHERE neutered = true AND escape_attempts < 3;
+  name
+---------
+ Agumon
+ Gabumon
+(2 rows)
```

Comment:
- [ ] It would be better not to comment on the queries commands and they should executable in the SQL 
when I put them in the console to test commands I got an error please make sure to keep only the commands and comment the tables or remove them the screenshots in the PR are enough